lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
```

Comment:
Might consider requiring specifying an API version -- even if volapi doesn't do API versioning yet. Require users to specify "acceptVersion: ~1" or explicitly "acceptVersion: *" and it makes moving to api versioning in volapi much easier later.